### ISSUE TITLE
Quickstart for Scala 3

### ISSUE THREAD

**Issue by carlosedp** [quarkiverse/quarkus-github-action#268]
Based on the Java codestart, I've created a Scala 3 quickstart here:

https://github.com/carlosedp/scala-github-action

Let me know what's needed to make this a codestart if desired!

**Comment by gsmet** [comment]
Unfortunately, I’m not entirely sure if we can make a Scala 3 codestart. 

IIRC, language codestarts need some hooks in core and it’s Scala 2 that is supported there. 

That being said, I will have a look at your QuickStart and see if something can be done.

**Comment by carlosedp** [comment]
Yes, I believe so since I looked into the PR that merged the Kotlin kickstart and I think some stuff like the POM compiler parts are injected by the core...
Thanks! Let me know.
